ucsb-cs156-f24/team02-f24-00
# Pc commit edit page

Closes #97 

In this PR we replace the placeholder Commit edit page with a working edit page.

To test:

1. Visit the dokku deployment at https://team02-dev-pconrad.dokku-00.cs.ucsb.edu
2. Login
3. Navigate to the commits page.  If there isn't a commit listed there, create one.
4. Click the edit button
5. Make changes to each of the fields and click Update
6. See that your changes took effect.

Storybook: https://67217d09054c99a597a2d9ac-disqdsfnfg.chromatic.com/?path=/docs/pages-commits-commitseditpage--docs

Screenshot:

<img width="878" alt="image" src="https://github.com/user-attachments/assets/bf8ddaf6-50ac-4398-b8a6-ffa4f91b7206">
